tests/unit/branch_name_test.py

```diff
@@ -0,0 +1,71 @@
+import branch_name_verification_hook as branch_name_verification_hook
```

Comment:
this is already in master, so why is it being added in this PR? it has nothing to do with #229

Reply:
I have no idea why I have these files

Reply:
@PeterGhimself is branch_name_test.py the only file i should delete.  There are a bunch of other files that I didn't write that are in my PR for whatever reason such as import_test.py, branch_name_verification_hook.py, pytest.ini  ... Should I delete them all?

Reply:
`git diff f357b` to show where the issue happens. Im very confused to what happened and why GH says its a conflict

Reply:
@PeterGhimself @MartensCedric Cedric readded the module (thumbs up emoji)